pkg/api/types.go

```diff
@@ -607,6 +607,10 @@ type SecretVolumeSource struct {
 	// the volume setup will error. Paths must be relative and may not contain
 	// the '..' path or start with '..'.
 	Items []KeyToPath `json:"items,omitempty"`
+	// Mode bits to use on created files by default. The used mode bits will
+	// be the provided AND 0777.
+	// Directories within the path are not affected by this setting
```

Comment:
Add a line about the default behavior.  Specifically, we can:

a) define the default value
b) define that the default is implementation-defined

What do you think?  I think specifying the default makes a nicer API, but it means we need to actually set the default value in defaults.go.